sql/src/main/java/org/apache/druid/sql/calcite/aggregation/builtin/CountSqlAggregator.java

```diff
@@ -72,29 +75,41 @@ static AggregatorFactory createCountAggregatorFactory(
       final Project project
   )
   {
-    final RexNode rexNode = Expressions.fromFieldAccess(
```

Comment:
Why are changes to non-distinct `COUNT` needed?

If we are making these changes— should we have doc updates for them too?

Reply:
Added docs updates. It's additional syntax support, I thought I will add it as I was making changes to distinct count.